system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs

```diff
@@ -1187,6 +1189,7 @@ mod benches {
 		[pallet_xcm, PalletXcmExtrinsicsBenchmark::<Runtime>]
 		// Bridges
 		[pallet_xcm_bridge_hub_router, ToKusama]
+		[snowbridge_pallet_system_frontend, SnowbridgeSystemFrontend]
```

Comment:
@claravanstaden how do we generate `snowbridge_pallet_system_backend` weights for `asset-hub-polkadot/src/weights/snowbridge_pallet_system_backend.rs` used by:
```
	type BackendWeightInfo = weights::snowbridge_pallet_system_backend::WeightInfo<Runtime>;
```